libs/labelbox/src/labelbox/schema/ontology_kind.py

```diff
@@ -21,6 +21,30 @@ def get_ontology_kind_validation_error(cls, ontology_kind):
         return TypeError(f"{ontology_kind}: is not a valid ontology kind. Use"
                          f" any of {OntologyKind.__members__.items()}"
                          " from OntologyKind.")
+    
+    @staticmethod
+    def evaluate_ontology_kind_with_media_type(ontology_kind,
+                                               media_type: Optional[MediaType]) -> Union[MediaType, None]:
+        
+        if ontology_kind and ontology_kind is OntologyKind.ModelEvaluation:
+            if media_type is None:
+                media_type = MediaType.Conversational
+            else:
+                if media_type is not MediaType.Conversational:
+                    raise ValueError(
+                        "For chat evaluation, media_type must be Conversational."
+                    )
+        
+        elif ontology_kind == OntologyKind.ResponseCreation:
+            if media_type is None:
+                media_type = MediaType.Text
+            else:
+                if media_type is not MediaType.Text:
+                    raise ValueError(
+                        "For response creation, media_type must be Text."
+                    )
```

Comment:
this can be greatly simplified and the level of unnecessary `if` nesting can be removed. Quick gpt prompting gives a nice suggestion for this:

```python
ontology_to_media = {
    OntologyKind.ModelEvaluation: (MediaType.Conversational, "For chat evaluation, media_type must be Conversational."),
    OntologyKind.ResponseCreation: (MediaType.Text, "For response creation, media_type must be Text.")
}

if ontology_kind in ontology_to_media:
    expected_media_type, error_message = ontology_to_media[ontology_kind]

    if media_type is None:
        media_type = expected_media_type
    else:
        raise ValueError(error_message)
```

Reply:
Clever! Added

Reply:
I did add on to this:
```python
    if media_type is None or media_type == expected_media_type:
        media_type = expected_media_type
    else:
        raise ValueError(error_message)
```
 This is added for users who already gave the correct expected_media_type through how it is set up, but just in case.